src/components/Blog/Post/Markdown/styles.module.css

```diff
@@ -631,3 +631,50 @@
     }
   }
 }
+
+.admonition {
+  padding: 10px;
+  border-left-width: 3px;
+  border-left-style: solid;
+  margin-bottom: 10px;
+
+  &.warn {
+    border-color: var(--color-orange);
+    background-color: rgb(227 112 70 / 20%);
+
+    .title::before {
+      content: "⚠️";
```

Comment:
Just for the fact that they look nicer and more customizable to the design. I think once @yalozhkin looks into this we will have them as well. This should be good to go for now before we have a final design.

Reply:
> Just for the fact that they look nicer and more customizable to the design. I think once @yalozhkin looks into this we will have them as well. This should be good to go for now before we have a final design.

I like the idea of using custom images, but would it look better? The docs and blog tend to use regular emojis in their text. For consistency, would it look better to keep them as emojis?  

> I think we'll also want to bind icons to string presets instead of having editors import/add icons directly, both for easier changes and removing the possibility of adding duplicated icons (e.g. two different pages using different github icon files).

Good idea. I'm definitely for binding icons to a string :)

> We may want support for both, but if we could only choose one I'd say SVG- Emojis can have radically different styles between different machines considering OSes, browsers, and fonts.

This is true, but the docs and blogs already use different emojis that change across different machines. As I mentioned before, should we allow this for consistency?

Reply:
> > Just for the fact that they look nicer and more customizable to the design. I think once @yalozhkin looks into this we will have them as well. This should be good to go for now before we have a final design.
> 
> I like the idea of using custom images, but would it look better? The docs and blog tend to use regular emojis in their text. For consistency, would it look better to keep them as emojis?
> 
> > I think we'll also want to bind icons to string presets instead of having editors import/add icons directly, both for easier changes and removing the possibility of adding duplicated icons (e.g. two different pages using different github icon files).
> 
> Good idea! I'm definitely for binding icons to a string :)
> 
> > We may want support for both, but if we could only choose one I'd say SVG- Emojis can have radically different styles between different machines considering OSes, browsers, and fonts.
> 
> This is true, but the docs and blogs already use different emojis that change across different machines. As I mentioned before, should we keep this for consistency?

Emojis being used throughout the blog is a good point I hadn't considered. While I'd say text content is different from the admonition icons, we do use emojis like the light bulb in `details` summaries which is enough to say emojis are good enough to admonitions.

Reply:
> I like the idea of using custom images, but would it look better? The docs and blog tend to use regular emojis in their text. For consistency, would it look better to keep them as emojis?

Sure, if regular emojis can fulfill the requirements then it's totally fine. Agreeing with Roger, we can support both so that we can consider custom images for GitHub, GitLab, and so on. Not to deviate from root feature to nice to have feature, that can go on to the next iteration as well.